authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.adoc

```diff
@@ -81,6 +81,11 @@ data:
 <3> The resource URL of the service account that can be impersonated with these credentials.
 <4> The path to the service account token inside the pod. By convention, this is `/var/run/secrets/openshift/serviceaccount/token` for {product-title} components.
 
+[IMPORTANT]
+====
+In {product-title} 4.10.8, support for using GCP Workload Identity for the image registry is removed from {product-title} 4.10, and you must configure the image registry to use long-lived credentials. This mitigation is planned to last until Workload Identity support for the image registry is restored in a later release.
```

Comment:
I might suggest rewording this to (as it was a bit confusing and it would be nice if we could give a reason why):

In {product-title} 4.10.8, image registry support is removed when you use GCP Workload Identity [due to ???]. To use the image registry in {product-title} 4.10.8, you must configure the image registry to use long-lived credentials instead. This mitigation is planned to last until Workload Identity support for the image registry is restored in a later release.

Reply:
Tweaked a tiny bit, will push the update shortly. Thanks!